molgenis-data-mapper/src/main/resources/js/single-mapping-project.js

```diff
@@ -52,17 +52,20 @@
 	        var currentForm = this;
 	        bootbox.confirm("Are you sure?", function(result) {
 	            if (result) {
+	            	showSpinner();
 	                currentForm.submit();
 	            }
 	        });
 	        return false;
 	    });
 		
-		$('#submit-new-source-column-btn').click(function() {
-			$('#create-new-source-form').submit();
+		$('#submit-new-source-column-btn').on('click', function(e) {
+			showSpinner();
+	        $('#create-new-source-form').submit();
```

Comment:
On Safari, I can still click the submit button twice before submission if I'm fast enough. I still get the json error message.

But now if I then want to get back to the menu so I push the back button on the empty json page, I see the previous page with a spinner that I cannot click away.